src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -2787,9 +2787,9 @@
     <Field Name="UsingKeyword" Type="SyntaxToken">
       <Kind Name="UsingKeyword"/>
     </Field>
-    <Choice>
-      <Field Name="StaticKeyword" Type="SyntaxToken" Optional="true"/>
-      <Field Name="Alias" Type="NameEqualsSyntax" Optional="true"/>
+    <Choice Optional="true">
+      <Field Name="StaticKeyword" Type="SyntaxToken"/>
+      <Field Name="Alias" Type="NameEqualsSyntax"/>
     </Choice>
```

Comment:
this has no impact on the generated .cs files. but it helps make things cleaner for the .g4 grammar file.